src/widgets/medications/medication-level-three/medication-level-three.component.tsx

```diff
@@ -0,0 +1,216 @@
+              <span style={{ display: "block" }}>NO </span>
+            </div>
+          </SummaryCardRowContent>
+          <SummaryCardRowContent justifyContent="space-between">
+            <div style={{ float: "left" }}>
+              <span style={{ color: "var(--omrs-color-ink-low-contrast)" }}>
+                {" "}
+                End date
+              </span>
+              <span style={{ display: "block" }}>
+                {" "}
+                {dayjs(patientMedication.dateStopped).format(
+                  "DD-MMM-YYYY"
+                )}{" "}
+              </span>
+            </div>
+            <div
+              style={{
+                float: "right"
+              }}
+            >
+              <span style={{ color: "var(--omrs-color-ink-low-contrast)" }}>
+                Dosing Instructions
+              </span>
+              <span style={{ display: "block" }}>
+                {patientMedication.drug.strength}
+                {" \u2014 "}
+                {patientMedication.frequency.display}{" "}
+              </span>
+            </div>
+          </SummaryCardRowContent>
+          <SummaryCardRowContent justifyContent="space-between">
+            <div style={{ float: "left" }}>
+              <span style={{ color: "var(--omrs-color-ink-low-contrast)" }}>
+                {" "}
+                Duration
+              </span>
+              <span
+                style={{
+                  display: "block"
+                }}
+              >
+                {" "}
+                {patientMedication.duration}
+                {patientMedication.durationUnits.display}{" "}
+              </span>
+            </div>
+          </SummaryCardRowContent>
+          <SummaryCardRowContent justifyContent="space-between">
+            <div style={{ float: "left" }}>
+              <span style={{ color: "var(--omrs-color-ink-low-contrast)" }}>
+                {" "}
+                Total number of refills
+              </span>
+              <span
+                style={{
+                  display: "block"
+                }}
+              >
+                {" "}
+                {patientMedication.numRefills}{" "}
+              </span>
+            </div>
+          </SummaryCardRowContent>
+        </SummaryCard>
+      </>
+    );
+  }
+  function details() {
+    return (
+      <SummaryCard
+        name="Details"
+        match={props.match}
+        styles={{
+          width: "90%",
+          backgroundColor: "var(--omrs-color-bg-medium-contrast)"
+        }}
+      >
+        <div className={`omrs-type-body-regular ${styles.medicationCard}`}>
+          <table className={styles.medicationTable}>
+            <thead>
+              <tr>
+                <td>Last updated</td>
+                <td>Last updated by</td>
+                <td>Last updated location</td>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td data-testid="last-updated">
```

Comment:
although `data-testid` is part of the `react-testing-library`, it's highly discouraged to use it because we are adding test attributes to the source code  that have no function aside from helping tests run. Say in future if we need to migrate away from the react-testing-library, it would involve a lot of work to change tests as well as some source code as well. I'd encourage you not to use it, and figure out a different way. One way would be to get an array of this row's children and use plain old indexes to access the indexes. cc @donaldkibet

Reply:
@kagai any updates on this PR?

Reply:
@jonathandick    ![screenshot](https://user-images.githubusercontent.com/14895251/74711161-5705db00-5234-11ea-9184-ebe455c67411.png)